examples/multi_blade_row_example.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2023 ANSYS, Inc. All rights reserved
+"""
+.. _multi_blade_row_example:
+
+Multi blade-row meshing example
+-------------------------------
+This basic example shows how to set up a multi blade-row meshing instance and execute it in parallel
+
+"""
+
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform the required imports. It is assumed that the ``ansys-turbogrid-core``
+# package has been installed.
+
+import os
+
+from ansys.turbogrid.core.multi_blade_row.multi_blade_row import MBR
+
+#################################################################################
+# Create and use an MBR instance
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create an MBR instance, set it up for a multi-blade row case and execute.
+
+if __name__ == "__main__":
+    # Set working directory:
+    working_directory = os.getcwd()
+
+    # Set location for the NDF file
+    case_folder = r"E:\AnsysDev\pyAnsys\S947493_concepts_nrec\turb_axial_4stage_geom"
+
+    # Set name of the NDF file
+    ndf_file_name = "turb_axial_4stage_geom.ndf"
+
```

Comment:
Make sure all the instructions have a period at the end (or none if that is preferred, but make sure that all are consistent).